src/lucky/errors.cr

```diff
@@ -17,9 +17,7 @@ module Lucky
   class ParamParsingError < Error
     include Lucky::RenderableError
 
-    getter request
-
-    def initialize(@request : HTTP::Request)
```

Comment:
Could this be added? The reason for it is that it can be used in `Errors::Show` or in future error pages to show the request body/url that could not be parsed